Split/Networking/SplitModels/SplitModel/Split.swift

```diff
@@ -22,4 +22,11 @@ import Foundation
     var trafficAllocationSeed: Int?
     var algo: Int?
     
+    var isValid: Bool {
+         return name != nil && seed != nil && killed != nil && defaultTreatment != nil &&
+                conditions != nil && trafficTypeName != nil && changeNumber != nil &&
+                trafficAllocation != nil && trafficAllocationSeed != nil &&
```

Comment:
trafficAllocationSeed based on the code from the BE could possible be null

Reply:
same as algo

Reply:
Updated